docs/configuration.md

```diff
@@ -203,7 +203,8 @@ Within each section, list the resources of the corresponding resource type along
 
     - `name` - specify the resource name to be monitored.
 
-        >  If you would like to monitor the resources with a common pattern in their names (e.g., glue-pipeline1-ingest, glue-pipeline1-cleanse, glue-pipeline1-staging), use wildcards: glue-pipeline1`-*`.
+        >  If you would like to monitor the resources with a common pattern in their names (e.g., glue-pipeline1-ingest, glue-pipeline1-cleanse, glue-pipeline1-staging), use wildcards: glue-pipeline1`-*`. \
+        > Note: Wildcards are not supported for Glue Data Quality Rulesets executed within Glue Jobs. The full name of the Ruleset should be provided.
```

Comment:
not critical, but I'm wondering if we change to "positively formulated" statements.
Like 
Note: In Glue DQ wildcards are supported only for Rulesets executed in .... For other Rulesets please specify an exact name (without wildcards).
Again, not critical, just polishing.

Reply:
sure, will update accordingly